public/app/percona/integrated-alerting/components/AlertRuleTemplate/AlertRuleTemplate.constants.ts

```diff
@@ -1,9 +1,16 @@
 export const ALERT_RULE_TEMPLATES_TABLE_ID = 'alert-rule-templates';
-import { TemplateParamUnit } from './AlertRuleTemplate.types';
+import { SourceDescription, TemplateParamUnit } from './AlertRuleTemplate.types';
 
 export const UNIT_MAP: Record<TemplateParamUnit, string> = {
   [TemplateParamUnit.PERCENTAGE]: '%',
   [TemplateParamUnit.SECONDS]: 'seconds',
 };
 
+export const SOURCE_MAP: Record<SourceDescription, string> = {
+  [SourceDescription.BUILT_IN]: 'Built-in',
+  [SourceDescription.SAAS]: 'Percona Enterprise Platform',
```

Comment:
@fabio-silva according to @shoffman-percona we should name those like this: 
```
SAAS: 'Percona',
USER_FILE: 'User Created (file)',
USER_API: 'User Created (UI)',
```

Reply:
in our case it should probably be:
`
PERCONA: 'Percona',
USER_FILE: 'User Created (file)',
USER_API: 'User Created (UI)',
`

Reply:
@atymchuk here we are mapping between the API's enum and what we show on the UI, so it got to be `SAAS`.
We can of course change our TS enum to be `PERCONA = 'SAAS'`, but the underlying value will still be the one from the API.